tests/Unit/Libs/Soa/v1/YousignTest.php

```diff
@@ -536,4 +544,222 @@ function (ClientRequest $request) use ($url): bool {
             $actualActivateSignatureResponse->toArray()
         );
     }
+
+    /**
+     * @test
+     * @covers      ::getSignatureById
+     *
+     * @return void
+     */
+    public function it_gets_signature_by_id(): void
+    {
+        /** @var string $url */
+        $url = Str::finish(
+            Config::get(YousignClientServiceProvider::CONFIG_KEY . '.url'),
+            Soa::URL_SEPARATOR
+        ) . Yousign::INITIATE_SIGNATURE_URL . Soa::URL_SEPARATOR . self::SIGNATURE_ID;
+
+        /** @var InitiateSignatureResponse $expectedSignatureResponse */
+        $expectedSignatureResponse = InitiateSignatureResponse::factory()
+                                                              ->make();
+
+        Http::fake(
+            [
+                $url => Http::response($expectedSignatureResponse->toArray(), Response::HTTP_CREATED),
+            ]
+        );
+
+        /** @var InitiateSignatureResponse|null $actualSignatureResponse */
+        $actualSignatureResponse = (new Yousign())->getSignatureById(self::SIGNATURE_ID);
+
+        Http::assertSent(
+            function (ClientRequest $request) use ($url): bool {
+                $this->assertSame(Request::METHOD_GET, $request->method());
+                $this->assertSame($url, $request->url());
+
+                $this->assertSame(
+                    Yousign::BEARER_PREFIX . Config::get(YousignClientServiceProvider::CONFIG_KEY . '.api_key'),
+                    Arr::first($request->header(Soa::AUTHORIZATION_HEADER))
+                );
+
+                $this->assertContains(
+                    Yousign::BEARER_PREFIX . Config::get(YousignClientServiceProvider::CONFIG_KEY . '.api_key'),
+                    $request->header(Soa::AUTHORIZATION_HEADER)
+                );
+
+                return true;
+            }
+        );
+
+        $this->assertNotNull($actualSignatureResponse);
+        $this->assertInstanceOf(InitiateSignatureResponse::class, $actualSignatureResponse);
+
+        $this->assertSame(
+            $expectedSignatureResponse->toArray(),
+            $actualSignatureResponse->toArray()
+        );
+    }
+
+    /**
+     * @test
+     * @covers      ::getDocumentById
+     *
+     * @return void
+     */
+    public function it_gets_document_by_id(): void
+    {
+        /** @var string $url */
+        $url = Str::finish(
+            Config::get(YousignClientServiceProvider::CONFIG_KEY . '.url'),
+            Soa::URL_SEPARATOR
+        ) . Yousign::INITIATE_SIGNATURE_URL . Soa::URL_SEPARATOR . self::SIGNATURE_ID . Soa::URL_SEPARATOR . Yousign::UPLOAD_DOCUMENT_URL . Soa::URL_SEPARATOR . self::DOCUMENT_ID;
+
+        /** @var string $expectedDownloadDocumentResponse */
+        $expectedDownloadDocumentResponse = YousignFaker::FAKE_IMAGE_STRING;
```

Comment:
Non hai bisogno di assegnare ad una variabile il contenuto di una costante che ti serve per fare i test (a meno che tu non debba poi alterarne il contenuto in qualche modo, ma non mi pare sia questo il caso).

Dove ti serve quell'array, usa pure la costante (vale per tutti i casi).